src/Standards/Generic/Docs/Commenting/DocCommentStandard.xml

```diff
@@ -0,0 +1,268 @@
+        <code title="Valid: DocBlock with a short description on the first line.">
+        <![CDATA[
+/**
+ * <em>Short description.</em>
+ */
+        ]]>
+        </code>
+        <code title="Invalid: DocBlock without a short description or short description not on the first line.">
+        <![CDATA[
+/**
+ * <em></em>@return int
+ */
+
+/**
+<em> *</em>
+ * Short description.
+ */
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    The short and long descriptions must start with a capital letter.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Short and long descriptions start with a capital letter.">
+        <![CDATA[
+/**
+ * <em>S</em>hort description.
+ *
+ * <em>L</em>ong description.
+ */
+        ]]>
+        </code>
+        <code title="Invalid: Short or long descriptions don't start with a capital letter.">
+        <![CDATA[
+/**
+ * <em>s</em>hort description.
+ *
+ * <em>l</em>ong description.
+ */
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    There must be exactly one blank line separating the short description, the long description and tag groups.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: One blank line separating the short description, the long description and tag groups.">
+        <![CDATA[
+/**
+ * Short description.
+<em> *<em>
+ * Long description.
+<em> *</em>
+ * @param int $foo
+ */
+        ]]>
+        </code>
+        <code title="Invalid: More than one or no blank line separating the short description, the long description and tag groups.">
+        <![CDATA[
+/**
+ * Short description.
+<em> *
+ *
+</em>
+ * Long description.
+ * @param int $foo
+ */
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    Parameter tags must be grouped together.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Parameter tags grouped together.">
+        <![CDATA[
+/**
+ * Short description.
+ *
+<em> * @param int $foo
+ * @param string $bar</em>
+ */
+        ]]>
+        </code>
+        <code title="Invalid: Parameter tags not grouped together.">
+        <![CDATA[
+/**
+ * Short description.
+ *
+ * @param int $foo
+<em> *</em>
+ * @param string $bar
+ */
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    Parameter tags must not be grouped together with other tags.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Parameter tags are not grouped together with other tags.">
+        <![CDATA[
+/**
+ * Short description.
+ *
+ * @param int $foo
+ *
+<em> * @author Some Author
+ * @return int</em>
+ */
+        ]]>
+        </code>
+        <code title="Invalid: Parameter tags grouped together with other tags.">
+        <![CDATA[
+/**
+ * Short description.
+ *
+<em> * @param  int $foo
+ * @author Some Author
+ * @return int</em>
+ */
```

Comment:
Same thing about the code sample:
```suggestion
/**
 * Short description.
 *
<em> * @param  int $foo
 * @since 5.4.8
 * @return int</em>
 */
```